include/deal.II/base/tensor.h

```diff
@@ -117,10 +117,14 @@ class Tensor<0,dim,Number>
    */
   Tensor ();
 
+#ifdef DEAL_II_WITH_CXX11
   /**
    * Copy constructor.
    */
-  Tensor (const Tensor<0,dim,Number> &initializer);
+  Tensor (const Tensor<0,dim,Number> &initializer) = default;
+  // always implicitly created, in case of C++11 we make this explicit (for
```

Comment:
I think it makes sense to move these comments into the doxygen string and maybe reformulate it a bit. Suggestion: "Implicitly generated copy constructor that copies the Tensor @p initializer element-by-element."